test/libsolidity/Metadata.cpp

```diff
@@ -226,6 +226,45 @@ BOOST_AUTO_TEST_CASE(metadata_stamp_experimental)
 		}
 }
 
+BOOST_AUTO_TEST_CASE(metadata_eof_experimental)
+{
+	// Check that setting an EOF version results in the experimental flag being set.
+	char const* sourceCode = R"(
+		pragma solidity >=0.0;
+		contract test {
+			function g(function(uint) external returns (uint) x) public {}
+		}
+	)";
+	for (auto metadataFormat: std::set<CompilerStack::MetadataFormat>{
+			 CompilerStack::MetadataFormat::NoMetadata,
+			 CompilerStack::MetadataFormat::WithReleaseVersionTag,
+			 CompilerStack::MetadataFormat::WithPrereleaseVersionTag
+		 })
```

Comment:
```suggestion
	for (auto metadataFormat: std::set<CompilerStack::MetadataFormat>{
		 CompilerStack::MetadataFormat::NoMetadata,
		 CompilerStack::MetadataFormat::WithReleaseVersionTag,
		 CompilerStack::MetadataFormat::WithPrereleaseVersionTag
	})
```

Reply:
Sorry, looks like I somehow got a mix of tabs and spaces there. I meant this:

```c++
	for (auto metadataFormat: std::set<CompilerStack::MetadataFormat>{
		CompilerStack::MetadataFormat::NoMetadata,
		CompilerStack::MetadataFormat::WithReleaseVersionTag,
		CompilerStack::MetadataFormat::WithPrereleaseVersionTag
	})
```

Reply:
Yeah, I should also have double-checked myself instead of just accepting :-). Should be fixed now.